### PR TITLE
make IsWearingGloves in core

### PR DESCRIPTION
**Pull request by darknight2590**
**Describe Pull request**
make IsWearingHandshoes in core

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

**Comment by darknight2590** [comment]
https://github.com/qbcore-framework/qb-core/pull/393

**Comment by Mojito-Fivem** [comment]
"Handshoes" is not English, you're also not filling in the PR template

**Comment by darknight2590** [comment]
@Mojito-Fivem yes only title im sorry
function QBCore.Functions.IsWearingGloves()

File: shared/main.lua [hunk 1]
```diff
@@ -74,4 +74,12 @@ QBShared.StarterItems = {
     ['phone'] = { amount = 1, item = 'phone' },
     ['id_card'] = { amount = 1, item = 'id_card' },
     ['driver_license'] = { amount = 1, item = 'driver_license' },
-}
\ No newline at end of file
+}
+
+QBShared.MaleNoHandshoes = {
```
Comment (izMystic) [inline shared/main.lua:79]: MaleNoGloves

File: shared/main.lua [hunk 2]
```diff
@@ -74,4 +74,12 @@ QBShared.StarterItems = {
...
+    [0] = true, [1] = true, [2] = true, [3] = true, [4] = true, [5] = true, [6] = true, [7] = true, [8] = true, [9] = true, [10] = true, [11] = true, [12] = true, [13] = true, [14] = true, [15] = true, [18] = true, [26] = true, [52] = true, [53] = true, [54] = true, [55] = true, [56] = true, [57] = true, [58] = true, [59] = true, [60] = true, [61] = true, [62] = true, [112] = true, [113] = true, [114] = true, [118] = true, [125] = true, [132] = true
+}
+
+QBShared.FemaleNoHandshoes = {
```
Comment (izMystic) [inline shared/main.lua:83]: FemaleNoGloves

**Pull request merged**